arc.tex

```diff
@@ -0,0 +1,110 @@
+In the case of a total wipe out of the \gls{OGA} systems use of Chef, docker etc allow redeployment rapidly.
+See also the \gls{SLAC} Rubin disaster recovery plan. \action{RD}{March 22}{Need a \gls{SLAC} disaster recover plan}.
+
+
+\subsection{ Offline Production  \gls{Enclave}}
+Each year (or more frequently), the Offline Production enclave takes the raw images accumulated to date in the Archive and reprocesses them to generate highly accurate, consistent images and measurement catalogs, known as a Data Release. These data products are stored in the Archive and made available to data rights holders in the DAC after they have been checked by automated systems and after Rubin Observatory staff has vetted, characterized, and documented them. Offline Production is split between the USDF and the \gls{FrDF} and UKDF. Each Data Facility performs part of the computations and exchanges its results with the others, so all have a complete set of data products at release time.
+
+
+\subsubsection{ Threats and Security infrastructure}
+\subsubsection{Disaster recovery}
+
+
+\subsection{ \gls{Archive}  Enclave}
+The raw images, data products, and other records of the survey such as commands, events, and telemetry from Observatory systems are all stored in the \gls{Archive}. As the permanent scientific record of the survey, no more than 1% of the raw images or telemetry may be lost or corrupted.
+
+To help ensure this, the French Data Facility maintains a disaster recovery copy of all raw images and data products. Additional copies of some raw images and data products will be stored in Observatory systems in Chile.
+
+\subsubsection{ Threats and Security infrastructure}
+\subsubsection{Disaster recovery}
+
+
+
+\subsection{ USDF \gls{Data Access Center} Enclave}
+Data rights holders will use the services and systems in this enclave to work with the survey data products.
+It is therefore a general-purpose scientific computing facility. Generally users will interact with the Rubin Science Platform (\gls{RSP}), which is composed of a web-based Portal Aspect providing a guided user interface for accessing and analyzing the data, a Notebook Aspect providing an interactive, flexible, programming-oriented interface, and an API Aspect providing an programmable access service.
+Users of the DAC may connect from anywhere in the world over the Internet; all such users will be authenticated before accessing any \gls{RSP} service.
+The \gls{RSP} is hosted on a cloud service, currently  Google Cloud Platform.
+
+The DAC retrieves the released data products from the \gls{Archive} Enclave via protocols and services authenticated at a service account level only. While end-user identities may be included for audit and accounting purposes, fundamentally the DAC exists to provide access to all \gls{Archive} contents.
+
+\subsubsection{ Threats and Security infrastructure}
+The \gls{RSP} is a major attack surface for the DAC.
```

Comment:
```suggestion
The \gls{RSP} is an attractive generic target due to its computing resources but has no confidential data.
```

Reply:
User-generated data is still sensitive.

Reply:
an attractive generic target due to its computing resources.
There is some user genereated data which is mildly sensitive.